cassandra-analytics-core/src/main/java/org/apache/cassandra/spark/data/ClientConfig.java

```diff
@@ -43,6 +43,14 @@ public final class ClientConfig
     public static final String DC_KEY = "dc";
     public static final String CREATE_SNAPSHOT_KEY = "createSnapshot";
     public static final String CLEAR_SNAPSHOT_KEY = "clearSnapshot";
+    /**
+     * snapshotTTL a time to live option for the snapshot (available since Cassandra 4.1+).
+     * TTL value specified must contain unit along. For e.g. 2d represents a TTL for 2 days;
+     * 1h represents a TTL of 1 hour, etc. Valid units are {@code d}, {@code h}, {@code s},
+     * {@code ms}, {@code us}, {@code µs}, {@code ns}, and {@code m}.
+     */
+    public static final String USER_PROVIDED_SNAPSHOT_TTL = "snapshot_ttl";
+    public static final String DEFAULT_SNAPSHOT_TTL = "2d";
```

Comment:
this seems like a reasonable default. I would expect most jobs would complete in well under 1 day